server/webapp/WEB-INF/rails.new/app/assets/javascripts/package_material_check_connection.js

```diff
@@ -15,18 +15,22 @@
  *************************GO-LICENSE-END**********************************/
 
 PackageMaterialCheckConnection = function (url) {
-
     _bind = function (parent_container_selector, check_connection_selector, connection_message_selector) {
         jQuery(parent_container_selector).on('click', check_connection_selector, function (e) {
             var connection_message_element = jQuery(connection_message_selector);
-            connection_message_element.removeClass("error_message").removeClass("ok_message").text('Checking connection...');
+            var formData = jQuery(parent_container_selector + " :input").serialize();
+            formData = formData.replace("_method=PUT", "_method=POST");
```

Comment:
yes.. otherwise the request will be "PUT" not post..
the same case in pluggableSCM
https://github.com/gocd/gocd/blob/master/server/webapp/WEB-INF/rails.new/app/assets/javascripts/pluggable_scm_check_connection.js#L24